### PR TITLE
Fixed HUD align bug in ammo history for ammo < 10 (#54)

### LINKED ISSUE #54: Aligning bug in HUD displaying ammo if amount is < 10
pictures talk for themselves

### PR DESCRIPTION
**Pull request by chinese-soup**
Fixes #54 

**Screenshots:**
image
image

### CODE REVIEW

File: cl_dll/ammohistory.cpp [hunk 1]
```diff
@@ -138,7 +138,9 @@ int HistoryResource :: DrawAmmoHistory( float flTime )
 				}
 
 				// Draw the number
-				gHUD.DrawHudNumberString( xpos - 10, ypos, xpos - 100, rgAmmoHistory[i].iCount, r, g, b );
+				char szString[32];
+				sprintf( szString, "%d", rgAmmoHistory[i].iCount );
```
Comment (YaLTeR) [inline cl_dll/ammohistory.cpp:142]: Just in case could you use `snprintf` here please?

**Review by YaLTeR — CHANGES_REQUESTED** [review]

File: cl_dll/ammohistory.cpp (hunk 1 above)
Comment (chinese-soup) [reply cl_dll/ammohistory.cpp]: For sure, it was a direct copy paste from DrawHudNumberString. Done.

Comment (chinese-soup) [reply cl_dll/ammohistory.cpp]: @YaLTeR I've noticed that the `DrawHudNumberString` function is no longer called by anything as it was only called in this instance, should I change it there instead?

Comment (YaLTeR) [reply cl_dll/ammohistory.cpp]: Yeah, that would make sense I guess.

Comment (chinese-soup) [reply cl_dll/ammohistory.cpp]: Never mind, it's used in `scoreboard.cpp` which isn't used _right now_, but could be if one were to implement this: https://github.com/YaLTeR/OpenAG/issues/89 so I'd say for compatibility, let's keep the DrawHudNumberString unchanged. Sorry about that.

Comment (YaLTeR) [reply cl_dll/ammohistory.cpp]: So this PR is good to go now right?

Comment (chinese-soup) [reply cl_dll/ammohistory.cpp]: Yep.

**Comment by YaLTeR** [comment]
Thanks!

**Pull request merged**